### ISSUE TITLE
Make it easier to have multiple executables

### ISSUE THREAD

**Issue by litchfld** [alcap-org/AlcapDAQ#78]
At the moment `main.o` is treated on the same footing as all other object files which is not ideal if you want to be able to build multiple applications, since all applications must define an `int main(argc argv)`.  It can be worked round but it's a bit of a nuisance.  

This showed up while I was trying to build a test harness, but it would be useful for other cases: building more complete development/regression tests; subsidary executables that create template files; etcetc...

**Comment by AndrewEdmonds11** [comment]
This has been done right, Phill?

**Comment by litchfld** [comment]
Nope.  I have hacked the makefile about a bit to make the unit test executable, but nothing systematic.  If we starting wanting a particular executable for calibration- or template-generating processes it might be helpful, but perhaps that just corresponds to specific card files.
